### ISSUE TITLE
Aux large influence the training phase.

### ISSUE THREAD

**Issue by YLongJin** [samleoqh/MSCG-Net#17]
It is great job.
when I traning my dataset, most of the time，the iou is satirfactory ，but sometimes the aux loss would  be very large. I debug the coed
I found the sgn input x which comput to gama would be abnormal.  I want to know why it will happend and can you give me some advice.
Thanks very much

**Comment by samleoqh** [comment]
it sounds like the the diagonal entries of A (adj matrix) are very small numbers in your case, then its `mean` value will be close to zero, thus the gama will be very large. A workaround is to use `clamp` to setup a maxum value to gama, e.g. adding code after # line #42 in scg_gcn.py 

`gama = torch.clamp(gama , min=1., max=5.0)`

**Comment by YLongJin** [comment]
I got it and the problem was solved, the input of the sgn is just numbered, how can I comput and get the min=1 max=5?
